lib/LaTeXML/Package/graphics.sty.ltxml

```diff
@@ -34,6 +35,27 @@ DefParameterType('GraphixDimension', sub {
       $gullet->readDimension; } },
   optional => 1);
 
+# For trim, viewport: a sequence of 4 dimensions
+# These must be space separated, but apparently trailing commas are ignored.
+DefParameterType('GraphixDimensions', sub {
+    my ($gullet) = @_;
+    $gullet->skipSpaces;
+    my @dims = ();
+    while (scalar(@dims) < 4) {
+      if (scalar(@dims)) {
+        my $t = $gullet->readToken;
+        if ($t && !$t->equals(T_OTHER(','))) {
+          $gullet->unread($t); } }
```

Comment:
minor aside: do we need a `$gullet->skipComma` method?

Reply:
possibly, but seems to be used slightly differently every time?